package.json

```diff
@@ -1,20 +1,15 @@
 {
   "license": "Apache-2.0",
+  "type": "module",
   "devDependencies": {
     "@playwright/test": "^1.44.0",
-    "@vuepress/bundler-webpack": "2.0.0-rc.9",
-    "@vuepress/plugin-docsearch": "2.0.0-rc.21",
-    "@vuepress/plugin-google-analytics": "2.0.0-rc.21",
-    "@vuepress/plugin-sitemap": "2.0.0-rc.21",
-    "@vuepress/theme-default": "2.0.0-rc.21",
-    "@vuepress/utils": "2.0.0-rc.9",
-    "sass-loader": "14.2.1",
-    "vue": "3.4.27",
-    "vuepress": "2.0.0-rc.9"
+    "@types/node": "^20.12.10",
+    "sass": "^1.77.0",
```

Comment:
this is probably no longer needed (consider we dont do customized vue component)